fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2219,6 +2219,25 @@ public class Config extends ConfigBase {
     @ConfField
     public static int experimental_lake_publish_version_threads = 16;
 
+    @ConfField(mutable = true)
+    public static int lake_autovacuum_max_previous_versions = 0;
+
+    @ConfField
+    public static int lake_autovacuum_parallel_partitions = 8;
+
+    @ConfField(mutable = true)
+    public static long lake_autovacuum_partition_naptime_seconds = 30;
```

Comment:
a little small, this may cause gc too frequent?

Reply:
Do you have any suggested value?

Reply:
I changed the default value to 180 seconds(3 minutes)